packages/edit-site/src/components/sync-state-with-url/use-init-edited-entity-from-url.js

```diff
@@ -54,7 +55,6 @@ function useResolveEditedEntityAndContext( { postId, postType } ) {
 			const {
 				getEditedEntityRecord,
 				getEntityRecords,
-				getDefaultTemplateId,
```

Comment:
If this PR is considered good, I'm going to include the removal of this selector/resolver/reducer as well since it's something recent (a new API not in core) that is not needed anymore.